### DISCUSSION TITLE
Pergunta sobre o 9º encontro em 19/11/2021.

### DISCUSSION THREAD

**Discussion by chicortiz** [clube-de-leitura-python/pense-em-python-2021#29] [Perguntas & Respostas]
Olá pessoal vamos mesmo revisitar o capítulo 7?

Um grande abraço a todos,

ChicO

**Comment by villares** [comment]
Salve! Acho que combinado foi seguir mesmo pro 8, eu devo ter colado o texto errado! arrumei!
